gamification/src/Components/CourseCard/CourseCard.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Button, Card } from "react-bootstrap";
+import { getCookie } from "../../Functions/Cookies";
+import Teacher from "./Dynamic/Teacher"
+
+function CourseCard() {
```

Comment:
receive dummy js object

Reply:
Need your thoughts on removing the dynamic components on Courses.js, to enable list rendering as I know it